pypeit/scripts/view_fits.py

```diff
@@ -68,19 +68,26 @@ def main(args):
             img = hdu[args.exten].data
         else:
             spectrograph = util.load_spectrograph(args.spectrograph)
+            bad_read_message = 'Unable to construct image.  In this use case, the code expects ' \
+                               'a raw image, not an output image product from pypeit.  To show ' \
+                               'a pypeit output image, specify the extension using --exten.  ' \
+                               'Use --list to show the extension names.'
             if args.proc:
-                # Use the arc FramePar since this does not do processing
-                par = spectrograph.default_pypeit_par()['calibrations']['arcframe']
-                par['process']['use_darkimage'] = False
-                par['process']['use_biasimage'] = False
-                par['process']['mask_cr'] = False
-                par['process']['cr_sigrej'] = -1
-
-                img = buildimage.buildimage_fromlist(spectrograph, int(args.det), par,
-                                                     [args.file]).image
+                # Use the biasframe processing parameters because processing
+                # these frames is independent of any other frames (ie., does not
+                # perform bias subtraction or flat-fielding)
+                par = spectrograph.default_pypeit_par()['calibrations']['biasframe']
+                try:
+                    img = buildimage.buildimage_fromlist(spectrograph, int(args.det), par,
+                                                        [args.file]).image
+                except:
+                    msgs.error(bad_read_message)
```

Comment:
If the above line fails it is not clear your error message is appropriate since it may have failed because the processing failed, but not because the user input the filename of the wrong type of image file.

Reply:
Good point.  I've modified the error message, including adding the original exception and message that caused the fault.